tox.ini

```diff
@@ -1,28 +1,17 @@
 [tox]
-envlist = py26,py27,pypy,py26-trunkdeps,py27-trunkdeps,pypy-trunkdeps
+envlist = py26-{twlatest,tw150},{py26,py27,pypy}-{twtrunk,twlatest,tw150}
```

Comment:
Yes, I think in the next Axiom release I will announce that as soon as Twisted releases with 2.6 support dropped, Axiom will be dropping support too.